### DISCUSSION TITLE
Images on appendix, become unformatted

### DISCUSSION THREAD

**Discussion by manuel-vinhas** [joaomlourenco/novathesis#492] [Q&A]
When adding a large number of images to the appendix, the section command does not format properly and some images are overlaid and not centered as well as their respective caption. I got some errors, one of which is "LaTeX Error: Too many unprocessed floats" on the command `\centering` on the image definition. 

![Captura de ecrã 2024-03-20 175007](https://github.com/joaomlourenco/novathesis/assets/101526088/ce985e1a-5ffd-4f68-9afb-c49cac4ea899)
![Captura de ecrã 2024-03-20 174959](https://github.com/joaomlourenco/novathesis/assets/101526088/642bd7bd-f084-4081-9e0e-dcead8bb737d)
![Captura de ecrã 2024-03-20 174945](https://github.com/joaomlourenco/novathesis/assets/101526088/88fc8124-c9a2-44ec-8799-f2aba4fb7479)
![image](https://github.com/joaomlourenco/novathesis/assets/101526088/0a67a95d-7e3b-4616-a10f-8c5a8b45961e)

**Comment by joaomlourenco** [comment]
I suggest you reduce the number of images to half, and check what happens.  If you still have an error, reduce to 1/4, and so on.  Once it works, add a few more and find when it fails.  Then, report the error messages you get.

**Reply by manuel-vinhas** [reply]
No matter the number of images, the declaration of the images right below `\section`, has not effect on the document formatation. The sections appear on the first page of the appendix, and all the images follow below. Despite the images being explicitly definide on the code below the `\section` command. Without a particular error. (First image) 

When adding one more image right before the errors appear, the new error that is showed is "LaTeX Error: Float(s) lost".

![image](https://github.com/joaomlourenco/novathesis/assets/101526088/16955a80-0e07-4366-826b-1f3c2cf03517)
![image](https://github.com/joaomlourenco/novathesis/assets/101526088/d4545582-2882-465b-a6c5-7d73a6bf3156)

**Comment by joaomlourenco** [comment]
Your figure envirnment should start with
```latex
\begin{figure}[htp]
```

**Reply by manuel-vinhas** [reply]
Now the images are on the right place, but the sections still are unformated. The images are not appearing below the respective sections.
![image](https://github.com/joaomlourenco/novathesis/assets/101526088/e4164927-39bc-4b1a-87c4-80d8ca67ed76)

**Reply by joaomlourenco** [reply]
Dear Manuel,
Please read a tutorial on LaTeX, learn something about LaTeX.  Then, do the things the right way.  Then, if something is wrong, please be VERY CLEAR on what is your problem.

What do you mean wirh _the sections still are unformated_???

**Reply by manuel-vinhas** [reply]
After i wrote some text at the beggining of the appendix, every image became displaced, again. Even with the `[htp]`. The errors are the same as before.
![image](https://github.com/joaomlourenco/novathesis/assets/101526088/b08a7237-db66-4004-8e0d-d3ae4d2bb048)
